### DISCUSSION TITLE
Will support for RSC be optional in the future?

### DISCUSSION THREAD

**Discussion by AsuraKev** [web-infra-dev/modern.js#4902] [General]
Hi there,

Just a question regarding the RSC as modern js plans to support it later on. Will modern js most likely to enable RSC by default or will it be an optional thing to have? Given that there are a lot of negative feedback about RSC..

thanks

**Comment by chenjiahan** [comment]
The support for RSC has not been designed yet, but I think RSC will be an optional feature in Modern.js, because Modern.js will always consider CSR users.
